### PR TITLE
use custom icons for list markers

### LINKED ISSUE #9783: Emoji not centered
I selected a user defined emoji for one of my cache lists (adventure labs):
Screenshot_20210111-222327_cgeo

On the map it seems those are not really centered...no technical issue, but does not look nice.
It might be possible, that the emoji is just not meant to be in the center...not sure?!

Screenshot_20210111-222345_cgeo

### PR DESCRIPTION
**Pull request by moving-bits**
Extending the support to cache list markers. 

Along the way some other things got fixed and/or added:
- add a generated set of colored circles (even on older Android devices not supporting the according Unicode characters)
- fixed vertical and horizontal alignment (fix #9783)
- created custom icons are cached
- existing assignments of list markers are migrated to their new counterparts
- removed circle symbols from first category in favor of the new "colored circles" category

|select list marker with new "colored circles" category:|custom list markers shown on map:|
|---|---|
|image|image|

### CODE REVIEW

File: main/res/layout/emojiselector.xml [hunk 1]
```diff
@@ -4,30 +4,27 @@
...
     android:layout_height="match_parent">
 
-    <!--
-    <LinearLayout
-        android:id="@+id/emoji_groups"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal" />
-
-    -->
```
Comment (fm-sys) [inline main/res/layout/emojiselector.xml:14]: Very cool, I really like it! Only one little thing, please remove this comment if the code is not needed anymore...

Comment (moving-bits) [reply main/res/layout/emojiselector.xml:14]: it's a leftover from an earlier PR and removed by this PR, so it's marked red

Comment (fm-sys) [reply main/res/layout/emojiselector.xml:14]: Oh, damn! I'm so sorry. I thought it was added in the PR... It was probably already too late last night 😅

Comment (moving-bits) [reply main/res/layout/emojiselector.xml:14]: No problem :-)

**Pull request merged**